src/ray/common/cgroup/cgroup_utils.cc

```diff
@@ -0,0 +1,163 @@
+// Copyright 2024 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "ray/common/cgroup/cgroup_utils.h"
+
+#include <sys/stat.h>
+
+#include <fstream>
+
+#include "absl/strings/str_format.h"
+#include "absl/strings/str_join.h"
+#include "absl/strings/str_split.h"
+#include "ray/util/logging.h"
+
+namespace ray {
+
+namespace {
+
+// Owner can read and write.
+constexpr int kCgroupFilePerm = 0600;
```

Comment:
can you replace all Cgroup to CgroupV2 ?

Reply:
sure, updated. but I would like to keep the filename.